CMakePresets.json

```
@@ -270,7 +271,17 @@
       "cacheVariables": {
         "CCCL_ENABLE_C": true,
         "CCCL_C_Parallel_ENABLE_TESTING": true,
-        "CCCL_C_Parallel_ENABLE_HEADER_TESTING": true
+        "CCCL_C_Parallel_ENABLE_HEADER_TESTING": true,
+        "CCCL_C_EXPERIMENTAL_STF_ENABLE_TESTING": true
```

Disable this here -- otherwise we'll test experimental in both the c-parallel and the c-stf jobs.

```suggestion
        "CCCL_C_EXPERIMENTAL_STF_ENABLE_TESTING": false
```